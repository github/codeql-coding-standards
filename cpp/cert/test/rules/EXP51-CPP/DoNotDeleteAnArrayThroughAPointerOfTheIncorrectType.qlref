rules/EXP51-CPP/DoNotDeleteAnArrayThroughAPointerOfTheIncorrectType.ql