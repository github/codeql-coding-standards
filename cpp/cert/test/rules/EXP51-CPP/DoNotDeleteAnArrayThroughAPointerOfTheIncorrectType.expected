edges
| test.cpp:6:19:6:37 | new[] | test.cpp:9:12:9:13 | l1 |
| test.cpp:7:22:7:40 | new[] | test.cpp:10:12:10:13 | l2 |
nodes
| test.cpp:6:19:6:37 | new[] | semmle.label | new[] |
| test.cpp:7:22:7:40 | new[] | semmle.label | new[] |
| test.cpp:9:12:9:13 | l1 | semmle.label | l1 |
| test.cpp:10:12:10:13 | l2 | semmle.label | l2 |
subpaths
#select
| test.cpp:9:12:9:13 | l1 | test.cpp:6:19:6:37 | new[] | test.cpp:9:12:9:13 | l1 | Array of type DerivedClass * is deleted through a pointer of type BaseClass *. |
